skimage/color/colorconv.py

```diff
@@ -500,6 +506,75 @@ def hsv2rgb(hsv, *, channel_axis=-1):
            'R': (1.0, 1.0, 1.0)}}
 
 
+def xyz_tristimulus_values(*, illuminant, observer, dtype=float):
+    """Get the CIE XYZ tristimulus values.
+
+    Given an illuminant and observer, this function returns the CIE XYZ tristimulus
+    values [2]_ scaled such that :math:`Y = 1`.
+
+    Parameters
+    ----------
+    illuminant : {"A", "B", "C", "D50", "D55", "D65", "D75", "E"}
+        The name of the illuminant (the function is NOT case sensitive).
+    observer : {"2", "10", "R"}
+        One of: 2-degree observer, 10-degree observer, or 'R' observer as in
+        R function ``grDevices::convertColor`` [3]_.
+    dtype: dtype, optional
+        Output data type.
+
+    Returns
+    -------
+    values : array
+        Array with 3 elements :math:`X, Y, Z` containing the CIE XYZ tristimulus values
+        of the given illuminant.
+
+    Raises
+    ------
+    ValueError
+        If either the illuminant or the observer angle are not supported or
+        unknown.
+
+    References
+    ----------
+    .. [1] https://en.wikipedia.org/wiki/Standard_illuminant#White_points_of_standard_illuminants
+    .. [2] https://en.wikipedia.org/wiki/CIE_1931_color_space#Meaning_of_X,_Y_and_Z
+    .. [3] https://www.rdocumentation.org/packages/grDevices/versions/3.6.2/topics/convertColor
+
+    Notes
+    -----
+    The CIE XYZ tristimulus values are calculated from :math:`x, y` [1]_, using the
+    formula
+
+    .. math:: X = x / y
+
+    .. math:: Y = 1
+
+    .. math:: Z = (1 - x - y) / y
+
+    The only exception is the illuminant "D65" with aperture angle 2° for
+    backward-compatibility reasons.
```

Comment:
I couldn't figure out where the original values for this case come from. There is 

https://github.com/scikit-image/scikit-image/blob/8495b071ff81f3f1c5c4d8a44cb0387a81092066/skimage/color/colorconv.py#L492

which points to 

https://github.com/scikit-image/scikit-image/blob/8495b071ff81f3f1c5c4d8a44cb0387a81092066/skimage/color/colorconv.py#L446-L448

I'm unsure where these values are derived from. A git blame investigation wasn't really fruitful yet. So for this PR I'd just ignore it...